control-plane-operator/hostedclusterconfigoperator/controllers/resources/olm/catalogs.go

```diff
@@ -5,33 +5,33 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
 	operatorsv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
 )
 
-func ReconcileCertifiedOperatorsCatalogSource(cs *operatorsv1alpha1.CatalogSource) {
-	reconcileCatalogSource(cs, "certified-operators:50051", "Certified Operators", -200)
+func ReconcileCertifiedOperatorsCatalogSource(cs *operatorsv1alpha1.CatalogSource, p *OperatorLifecycleManagerParams) {
+	reconcileCatalogSource(cs, "certified-operators:50051", p.CertifiedOperatorsImage, "Certified Operators", -400, p.OLMCatalogPlacement)
```

Comment:
/hold
@hasueki is there a reason you changed the priority for this one? (should be -200)